doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -411,6 +412,30 @@ If we choose to designate all variables as either design or state variables, wit
   >>> print("PyROS termination condition: %s" % results_1.grcs_termination_condition)
   PyROS termination condition: grcsTerminationCondition.robust_optimal
 
+PyROS Results Object
+"""""""""""""""""""""""""""
+The results object returned by PyROS allows you to query the following information from the solve call:
+total iterations of the algorithm ``iterations``, CPU time ``time``, the GRCS algorithm termination condition ``grcs_termination_condition``,
+and the final objective function value ``final_objective_value``. If the option ``load_solution`` = ``True`` (default), the variables in the model will be
+loaded to the solution determined by PyROS and can be obtained by querying the model variables.
+
+.. note::
+    The reported ``final_objective_value`` and final model variable values depend on the selection of the option ``objective_focus``.
+    The ``final_objective_value`` is the sum of first-stage and second-stage objective functions.
+    If ``objective_focus = ObjectiveType.nominal``, second-stage objective and variables are evaluated at the nominal realization of the uncertain parameters, :math:`q^0`.
+    If ``objective_focus = ObjectiveType.worst_case``, second-stage objective and variables are evaluated at the worst-case realization of the uncertain parameters, :math:`q^{k^\ast}` where :math:`k^\ast = argmax_{k \in \mathcal{K}} f_2(x,z^k,y^k,q^k)` .
+
+An example of how to query these values on the previously obtained results is shown in the code block below.
+
+.. code-block::
```

Comment:
Can this be a `.. doctest::` block?  None of this should output anything (so the example doesn't change), but by making it a doctest, the code is actually run (and testing that those field names are indeed correct)?

Reply:
Yes I can do that.